metricbeat/docker-compose.yml

```diff
@@ -91,3 +105,8 @@ services:
 
   zookeeper:
     image: jplock/zookeeper:3.4.8
+
+volumes:
```

Comment:
Same as above :)
I'll study the rest api but I think that we can remove all this and don't use command any more.